### DISCUSSION TITLE
VirusTotal warnings

### DISCUSSION THREAD

**Discussion by juiceb0xplus** [Ch4r0ne/ARK-Ascended-Server-Manager#23] [General]
Is this script safe? https://www.virustotal.com/gui/file/264a3bd045b0381ae38f50260e87e79e343aff846b8690d64164eb770b32e9c2?nocache=1

**Comment by juiceb0xplus** [comment]
Just FYI Windows Defender started blocking downloads and quarantining the .exe file

**Reply by Ch4r0ne** [reply]
of course the skirpt is safe, unfortunately we had the problem before, I think the skipt just does too much automatically, which leads to a false positive.

I'd better not build an .exe anymore and write a detailed guide on how to use a .ps1.
